components/announcements/cta.js

```diff
@@ -21,7 +21,7 @@ export default function SlackCTA() {
           </Heading>
           <Text variant="subtitle" sx={{ lineHeight: 'caption', mb: 3 }}>
             Hack Club is a global community of high school makers & student-led
-            coding clubs. We’ve got a 24/7 Slack chatroom of 15k+ teenagers
+            coding clubs. We’ve got a 24/7 Slack chatroom of 17k+ teenagers
```

Comment:
<img width="447" alt="Screen Shot 2022-03-14 at 09 10 37" src="https://user-images.githubusercontent.com/72365100/158213778-9f48ca50-7209-4f65-aa2e-65ac5ff899be.png">
👁️

Reply:
Sure thing! Is this ready to merge? Everything else lgtm!

Reply:
Thank you so much Ella! I think so? Zach signed off on the copy before I made the PR :)

Reply:
No problem! Just wanted to check cause the date seems to be pre-dated (March 16), and I don't think the money has been received in Bank <img src="https://emoji.slack-edge.com/T0266FRGM/bank-hackclub/a72256ba735033c3.png" width="20">

Reply:
Oh wait, yes! I predated it the 16th as I wasn't sure when the announcement will be shared. Since it's kinda late EDT now, I assume it'll be tomorrow? I'll check with Christina! Thanks.

Reply:
Christina says it'll go out tomorrow ( 15 March ), I made the edits :) Thank you!

Reply:
Sounds good! I'll go ahead and merge.